src/icons/svg/hedgedoc.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="1536" height="1792" viewBox="0 0 1536 1792" xmlns="http://www.w3.org/2000/svg">
+    <path d="M 768.26562,124.11133 689.99023,202.39453 596.25391,143.5 537.35742,237.22852 432.86719,200.66211 396.30664,305.15234 286.29297,292.75586 273.90234,402.75977 163.89648,415.1543 l 12.39649,110.0039 -104.503908,36.56836 36.562498,104.48633 -93.732419,58.89453 58.902343,93.75 -78.2910152,78.28516 78.2832032,78.29297 -58.894531,93.74025 93.732419,58.8926 -36.562498,104.4882 104.499998,36.5684 -12.39258,109.9922 110.00586,12.4004 12.39063,110.0039 110.01367,-12.3906 36.56055,104.4804 104.48633,-36.5468 58.90039,93.7324 93.50586,-58.7617 c 0.43013,42.912 35.33333,77.5703 78.35156,77.5703 43.03103,0 77.94637,-34.6905 78.35547,-77.6114 l 0.082,-0.08 93.73242,58.8925 58.89258,-93.7421 104.5,36.5664 36.5567,-104.4883 110.0136,12.3886 12.3907,-110.0136 110.0136,-12.3906 -12.4004,-110.002 104.5,-36.5684 -36.5664,-104.5 93.7325,-58.8925 -58.8926,-93.72855 78.2832,-78.29297 -78.2832,-78.2793 58.8926,-93.72851 -93.7325,-58.90039 36.5664,-104.4961 -104.4902,-36.55859 12.3906,-110.00977 -110.0019,-12.39258 -12.4024,-110.01367 -110.0019,12.39258 -36.5684,-104.49414 -104.47852,36.5625 -58.91406,-93.73438 -93.74023,58.90039 z M 504.90234,745.42578 c 82.15511,0 155.81422,36.21661 206.02735,93.50391 l 57.07422,57.07422 69.80078,-69.80664 0.0351,0.0352 c 49.49011,-49.01259 117.56799,-79.29687 192.72071,-79.29687 151.3042,0 273.9571,122.65804 273.9571,273.9589 0,83.3298 -40.9567,156.3914 -95.9688,208.1681 l -362.31447,362.5488 c -20.03779,-19.9937 -47.64303,-32.3515 -78.11915,-32.3515 -30.42934,0 -57.97702,12.3172 -78.00976,32.2461 L 318.03906,1219.6367 c -53.56711,-49.9978 -87.0957,-121.1876 -87.0957,-200.2519 0,-151.30108 122.65276,-273.95902 273.95898,-273.95902 z m 111.79297,312.78512 c -33.06865,0 -59.87695,26.8086 -59.87695,59.8907 0,33.0822 26.8083,59.8906 59.87695,59.8906 33.08634,0 59.89258,-26.8084 59.89258,-59.8906 0,-33.0821 -26.80624,-59.8907 -59.89258,-59.8907 z m 303.7168,0 c -33.07178,0 -59.88867,26.8086 -59.88867,59.8907 0,33.0822 26.81689,59.8906 59.88867,59.8906 33.08217,0 59.89062,-26.8084 59.89062,-59.8906 0,-33.0821 -26.80845,-59.8907 -59.89062,-59.8907 z m -280.95703,34.3086 c 9.15898,0 16.59178,7.4382 16.5918,16.6035 0,9.1756 -7.43282,16.5821 -16.5918,16.5821 -9.16313,0 -16.60156,-7.4065 -16.60156,-16.5821 0,-9.1653 7.43843,-16.6035 16.60156,-16.6035 z m 303.70898,0 c 9.15482,0 16.5918,7.4382 16.5918,16.6035 0,9.1756 -7.43698,16.5821 -16.5918,16.5821 -9.16522,0 -16.60351,-7.4065 -16.60351,-16.5821 0,-9.1653 7.43829,-16.6035 16.60351,-16.6035 z" clip-rule="evenodd" fill-rule="evenodd" stroke-linejoin="round" stroke-miterlimit="2"/>
```

Comment:
`svgo` optimized `stroke-linejoin="round" stroke-miterlimit="2"` ( at the end of `<path>` ) on my local build. I am not a designer so I am not sure whether that is an actual optimization or a false positive. Do you think it is required for the SVG to work?

Reply:
This is what svgo produced when I ran make. :shrug: 
But no, these properties are not necessary. I removed them.

Reply:
It could be a version mismatch. I am running `svgo v1.3.2` which is what is in `package.json` https://github.com/ForkAwesome/Fork-Awesome/blob/master/package.json#L29